### ISSUE TITLE
Show only relevant buttons

### ISSUE THREAD

**Issue by jakub-bao** [pepfar-datim/datim-approvals#77]
don't show view on submit tab

don't show submit on view tab

**Comment by jakub-bao** [comment]
@benguaraldi We can do it. But it's a little complex.

I kinda like the idea which the old approvals app has. Instead of these buttons show checkboxes:

![image](https://user-images.githubusercontent.com/47790868/67585633-f4854e00-f71d-11e9-9734-e97c18915700.png)


Also it's consistent with the used case where you approve multiple mechanisms at once.

**Comment by benguaraldi** [comment]
@jakub-bao Yeah, I think checkboxes are fine.  Folks are used to them, for sure.

**Comment by jakub-bao** [comment]
I've got the mechanism select working and it returns list of selected mechanisms to its parent component.

However, it returns list as a set of objects ready to be rendered rather than the original mechanism model. Let's transform the data to the original mechanism model. It's much cleaner:

![image](https://user-images.githubusercontent.com/47790868/67590729-cc9be780-f729-11e9-801c-4d926ea15b26.png)
